camo.py

```diff
@@ -11,20 +11,17 @@ def __init__(self, server, key):
         self.key = key
 
     def image_url(self, url):
-        return self.server + Image(url, self.key).path
-
-    def _rewrite_url(self, url):
         # skip images that have already been proxied and
         # images with data URIs
         if url.startswith(self.server) or url.startswith("data:image/"):
```

Comment:
Shouldn't `or url.startswith("data:image/"):` go away if we are no longer allowing data URIs?

Reply:
Regardless of how we handle this I think we still need a test for it in the nbconvert_sanitize PR.

Reply:
Added a test for this in nbconvert-sanitize, https://github.com/quantopian/nbconvert-sanitize/pull/15/files#diff-0eae6505faaa25b2a42f9b0a8f6cbfd2R604